### ISSUE TITLE
Starting today, id.twitch.tv responds 404 Not Found

### ISSUE THREAD

**Issue by undeference** [twitchdev/issues#862]
**Brief description**
Any request to https://id.twitch.tv/oauth2/token responds with 404 Not Found

**How to reproduce**
Try to auth an account, wait for revalidation, anything

**Expected behavior**
It should work

**Screenshots**
![very useful omg screenshot](https://github.com/twitchdev/issues/assets/3173583/63b59df7-d275-4fe3-9841-ce7d7d9fc13c)
(Actual console output from my application)

**Additional context or questions**
It worked yesterday and I changed nothing

[Barry says "works for me"](https://discuss.dev.twitch.com/t/api-errors-oauth/50824/)

**Comment by jbulava** [comment]
Hi @undeference, I cannot reproduce this issue. My tests to refresh a token are succeeding.  Are you still experiencing this issue?  Is it just for refreshing a token, or any use of id.twitch.tv?  Can you provide more information, perhaps your location? You can also email developers@twitch.tv if there is any information you do not want to share publicly.

**Comment by undeference** [comment]
I am still experiencing this issue. It is not only for refreshing a token but also issuing a new token. As far as I know, it only seems to be /oauth2/token.

**Comment by undeference** [comment]
Hi,

Apparently I did introduce a bug on my testing system which resulted in the request being sent improperly (yes, as a GET). At the same time, my production system got disconnected and was inexplicably requiring a new token. I've never seen that before, so I reasonably but incorrectly assumed these were connected.

I fixed the bug in my test code and got that working, and I restarted the production instance and it's working happily now (with the old token).
